lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -8,6 +8,11 @@ module Style
       # This cop identifies places where `do_something(*args, &block)`
       # can be replaced by `do_something(...)`.
       #
+      # In Ruby 3.2, anonymous args/kwargs forwarding has been added.
+      #
+      # This cop identifies places where `use_args(*args)`/`use_kwargs(**kwargs)`
```

Comment:
Probably you should mention the config option controlling this behavior here, otherwise people might think that's something that's always getting checked.